src/rule.jl

```diff
@@ -316,6 +316,38 @@ macro rule(expr)
     end
 end
 
+"""
+    @capture ex LHS
```

Comment:
```suggestion
    @capture ex pattern
```
I think instead of calling it LHS here, it'd be better to call it `pattern`, since there isn't really a RHS!